clrfeatures.cmake

```diff
@@ -1,5 +1,5 @@
 if(CLR_CMAKE_TARGET_TIZEN_LINUX)
-  set(FEATURE_EVENT_TRACE 0)
+  set(FEATURE_EVENT_TRACE 1)
```

Comment:
If you want to enable this feature, you can simply remove this line. 
`FEATURE_EVENT_TRACE` is set by default on Linux/ARM32.

Reply:
@hseok-oh Thank you for suggestion, but I think it's more reliable  to explicitly set feature on.